test/bench/suites/string_deserialization.ts

```diff
@@ -18,7 +18,11 @@ export function getStringDeserializationSuite(BSON): Suite {
       data: DOC,
       fn: serializedDoc =>
         BSON.deserialize(serializedDoc, { validation: { utf8: utf8Validation } }),
-      iterations: 1000
+      iterations: 1000,
```

Comment:
I think 1000 is quite small. I'm not sure what is a good value to raise it to, probably north of 100,000.

Reply:
Raising it to 100,000 makes evergreen time out. Do we want to extend the timeout duration or reduce the number of iterations?

Reply:
Let's follow up when we have more benchmarks, I don't think we have an accurate data point on where JS will turn this code into machine instructions to decide here.